### DISCUSSION TITLE
Charge Count Text is Under Cooldown Texture in Alpha

### DISCUSSION THREAD

**Discussion by Brkymrky** [SFX-WoW/Masque#193] [Issues] [answered]
hello, I'm using the latest alpha version. my skin has a custom cooldown texture that has to be displayed on top of the border (normal texture). retail version had problems with this but I fixed it by changing Button.__MSQ_Cooldown:SetFrameLevel(base - 1) to there is Button.__MSQ_Cooldown:SetFrameLevel(base + 0).

eventually I had to switch because of circular charge texture in the alpha. now everything works great, except my charge count stays under the cooldown texture. so if possible I'd like to change it OR find a way to use latest build's circular textures in retail version. both works for me

**Comment by StormFX** [comment]
The reason the charge count appears below the `Cooldown` is because it uses the `Count` font string, which is parented to the button, not the `ChargeCooldown`. As far as the circular textures go, you can always just create a custom swipe texture with a circular alpha channel.

**Comment by Brkymrky** [comment]
thank you for answering my question. I'm new to all of this so I hope I'm not missing anything. this isn't a big issue but I'd still like to fix it, I'm kinda obsessive about these.  can I make an edit to lua and make it stay on top?

about the second part, my cooldown texture is circular, I noticed you have a new edge texture, can I somehow use that in the retail?

**Comment by StormFX** [comment]
> can I make an edit to lua and make it stay on top?

Not an easy one. As I said, the `Count` region is part of the Button frame. The `ChargeCooldown` is a separate frame.

> I noticed you have a new edge texture, can I somehow use that in the retail?

That would also take quite a bit of editing, as _Masque_ has to counter the game changing it. Also, the edge texture isn't round. That's accomplished via `:SetUseCircularEdge()`.

**Comment by Brkymrky** [comment]
got it. guess I won't be able to change the frame layers.

https://github.com/StormFX/Masque/issues/38

in this issue user zinoh mentioned using setusecircularedge with "viragdevtool". is this something the third users can do for personal use?

**Comment by StormFX** [comment]
You can look at _Masque_'s code and see if you can figure it out. :p

**Comment by Brkymrky** [comment]
I checked some files in the alpha version but whatever this viragdevtool is, I'm hearing it for the first time, so that's why I asked. 

already downloaded virag to try my chances but I have absolutely no idea what to do with it.  I assumed it would be easy for me to replicate since it's changing 1 setting in the code but everything becomes difficult when you have no experience really need some guidence here honestly, but I take it you are not familiar with this tool either (of if you are, would love to hear from you as well).

I'll message zinoh and ask for more information in the meantime. fingers crossed ^_^

**Comment by StormFX** [comment]
I don't know anything about "viragdevtool". I was speaking of editing _your local copy_.

**Comment by Brkymrky** [comment, marked as answer]
I see now. I'll try to tinker with settings. thanks a lot for your time man
